src/components/SidebarFilter/_sidebar-filter.scss

```diff
@@ -1,5 +1,15 @@
 // .cc-sidebar-filter {}
 
+.cc-sidebar-filter__body {
+  .cc-accordion__button {
+    font-size: var(--body-md);
+  }
+
+  .cc-accordion__button .icon {
+    color: var(--colour-black);
+  }
```

Comment:
An accordion as a filter has different styles: smaller font-size, black plus icon, updated styles for checkoboxes (see: https://github.com/wellcometrust/corporate-components/pull/437/files#diff-f7e6b45f11fc8c5b6939f03c7dd6746e385c590188bb9cd9808f2abdb4b276f6R78).

Maybe a better idea is to add a prop to the Accordion component?

Reply:
Yes I agree. A string union would be a good type to use to allow for other variants in future i.e.

```
// type definition
variant: 'standard' | 'filter';
// ...

// prop
variant = 'standard',
// ...
```